packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -909,11 +912,15 @@ class RectangularSliderTrackShape extends SliderTrackShape {
     }
 
     // Used to create a gap around the thumb iff the slider is disabled.
+    // If the slider is enabled, the track can be drawn beneath the thumb
+    // without gaps. But when the slider is disabled, the track is shortened
```

Comment:
Since, for now, there's only one thumb, I guess that's "without a gap".

Reply:
Done